### ISSUE TITLE
KVM test environement fails

### ISSUE THREAD

**Issue by Vincinator** [gardenlinux/gardenlinux#1626]
**What happened:**

- `make aws`
- configuring the test environment as described in [tests/README.md](https://github.com/gardenlinux/gardenlinux/tree/main/tests#readme)
```
 sudo podman run -it --rm --device=/dev/kvm -v /boot/:/boot -v /lib/modules:/lib/modules -v `pwd`:/gardenlinux -v ~/config:/config gardenlinux/base-test:`bin/garden-version` /bin/bash
```
```
pytest --iaas=kvm --configfile=/config/mykvmconfig.yaml
```
pytests can not connect with ssh to vm (started via  `start-vm`). 

**Possible solution** 
- Issue is https://github.com/gardenlinux/gardenlinux/blob/main/bin/start-vm#L601 
    - removing this option works

**Comment by Vincinator** [comment]
I had an typo in the ssh key path. Maybe we should add an early check.
Still testing before closing this issue again.

**Comment by Vincinator** [comment]
solved with #1627
